e2e/cypress/integration/bot_accounts/crud_spec.js

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @bot_accounts
+
+import {getRandomId} from '../../utils';
+
+describe('Bot accounts - CRUD Testing', () => {
+    let newTeam;
+    let botName;
+    let botDescription;
+    beforeEach(() => {
+        botName = 'bot-' + Date.now();
+        botDescription = 'test-bot-' + Date.now();
+
+        // # Set ServiceSettings to expected values
+        const newSettings = {
+            ServiceSettings: {
+                EnableBotAccountCreation: true,
+                DisableBotsWhenOwnerIsDeactivated: true,
+            },
+            PluginSettings: {
+                Enable: true,
+                RequirePluginSignature: false,
+            },
+        };
+        cy.apiUpdateConfig(newSettings);
+
+        // # Create a test bot
+        cy.apiCreateBot(botName, 'Test Bot', botDescription);
```

Comment:
Wonder if it is strictly needed to create a bot for each test. Can't we reuse the bot for different tests?

Reply:
can't reuse the bot, it's needed fresh and consistent in all of the tests